tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -97,9 +97,9 @@ def testCombinedChecksInstanceAndGraph(self, mock_entity, mock_gv, mock_iv):
     validator = entity_instance.CombinationValidator(self.config_universe,
                                                      _UPDATE_CFG, {})
 
-    self.assertTrue(validator.Validate(mock_entity))
-    mock_iv.assert_called_once_with(mock_entity)
-    mock_gv.assert_called_once_with(mock_entity)
+    self.assertTrue(validator.Validate(mock_entity()))
+    mock_iv.assert_called_once_with(mock_entity.return_value, False)
+    mock_gv.assert_called_once_with(mock_entity.return_value)
```

Comment:
with true/false?

Reply:
we understood this as ensuring that we called the mock with False and not True for the udmi flag.
the change ensures it was called with is_udmi= False